packages/@storefront/structure/src/carousel/index.ts

```diff
@@ -28,6 +28,13 @@ class Carousel {
 
   onMount() {
     utils.WINDOW().addEventListener('resize', this.forceUpdate);
+
+    // this forces an update on mount which in turn sets
+    // items to state from props by calling cloneItems
+    // so that items display within the carousel. 
+    // We are doing this because on mount of gb-carousel, it is 
+    // not populating the carousel with the item data contained in state.
```

Comment:
The reason for doing `this.set(true)` here instead of simply setting the items in `init` is because styles need to be recalculated to take into account the number of items.

Consider something like this:

```js
// the items have a width of 0 if items are simply set on init, so do an update to recalculate styles
```